libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/DropInNavigationViewContext.kt

```diff
@@ -69,12 +66,23 @@ internal class DropInNavigationViewContext(
     )
 
     //endregion
+
+    fun applyCustomization(action: Customization.() -> Unit) {
+        val c = Customization().apply(action)
+        uiBinders.applyCustomization(c)
+
+        c.routeLineOptions?.also { routeLineOptions = it }
+        c.routeArrowOptions?.also { routeArrowOptions = it }
+    }
 }
 
 /**
  * Helper extension to map [UIBinder] inside a [UICoordinator].
  * Uses a distinct by class to prevent refreshing views of the same type of [UIBinder].
  */
 internal fun <T : UIBinder> DropInNavigationViewContext.flowUiBinder(
-    mapper: suspend (value: NavigationUIBinders) -> T
-): Flow<T> = this.uiBinders.map(mapper).distinctUntilChangedBy { it.javaClass }
+    selector: (value: NavigationUIBinders) -> StateFlow<T>,
+    mapper: suspend (value: T) -> T = { it }
+): Flow<T> {
+    return selector(this.uiBinders).map(mapper).distinctUntilChangedBy { it.javaClass }
```

Comment:
It would be great to remove this `distinctUntilChangedBy { it.javaClass }`. To make it distinctBy class was a hack

`this.uiBinders.map(mapper)` should work now that each uiBinder has an individual stateFlow